common/src/main/java/org/dash/wallet/common/util/MonetarySpannable.java

```diff
@@ -50,7 +50,7 @@ private static CharSequence format(final MonetaryFormat format, final boolean si
         if (monetary == null)
             return "";
 
-        checkArgument(monetary.signum() >= 0 || signed);
+        //checkArgument(monetary.signum() >= 0 || signed);
```

Comment:
Why is this commented out?

Reply:
Hmm... It was commented out by @sambarboza , I don't know why.

Reply:
The checkArgument is checking for positive numbers, but we are now using this method to format negative numbers.